packages/block-editor/src/store/actions.js

```diff
@@ -811,6 +811,15 @@ export function __unstableMarkLastChangeAsPersistent() {
 	return { type: 'MARK_LAST_CHANGE_AS_PERSISTENT' };
 }
 
+/**
+ * Returns an action object used in signalling that the next block change should be marked explicitly as not persistent.
+ *
+ * @return {Object} Action object.
+ */
+export function __unstableMarkNextChangeAsNotPersistent() {
+	return { type: 'MARK_NEXT_CHANGE_AS_NOT_PERSISTENT' };
```

Comment:
To mark a change as persistent, we use MARK_LAST_CHANGE_AS_PERSISTENT to mark the previous change as persistent, and we trow the action after the action we want to mark. While to explicitly mark a change as not persistent, we use MARK_NEXT_CHANGE_AS_NOT_PERSISTENT, and we trow it before the action we want to mark.

Would it be possible to make the usage of the actions more similar and trow both actions before or after the actions we want to mark?

Reply:
No, because of usage nuance:

In our framework, when you want a persistent change to not be persistent, it's so that the non-dirtying change handler is called from the block editor. If you fire an edit, and then fire a `MARK_NEXT_CHANGE_AS_NOT_PERSISTENT`, nothing guarantees that the wrong change handler wasn't already called before the `MARK_NEXT_CHANGE_AS_NOT_PERSISTENT`.

`MARK_LAST_CHANGE_AS_PERSISTENT` is different, because when using it, it's ok that the non-dirtying handler might have already been called, it will just call the dirtying one again with the same values.